js/util/trumbowyg/hyperlink.js

```diff
@@ -0,0 +1,87 @@
+(function ($) {
+	'use strict';
+	$.extend(true, $.trumbowyg, {
+	langs: {
```

Comment:
that is a utility i created for making hyperlink compatible with email. default hyperlink in the dependency contained target and title so i had to extend it